scripts/check-tags.ts

```diff
@@ -38,16 +49,24 @@ const checkProjects = async () => {
   const projects = globSync('{apps,libs}/**/project.json', {
     ignore: '**/node_modules/**',
   })
-  let hasError = false
+  let invalidProjects = []
   for (const project of projects) {
-    hasError = hasError || (await checkTags(project))
+    // console.log("Checking project", project)
+    // Old
+    // hasError = hasError || (await checkTags(project))
+
+    // New, and faster
+    if (!(await hasValidTags(project).catch(() => false))) {
```

Comment:
Should not swallow errors. 

Which errors do you want to ignore? Can you handle them inside hasValidTags?

Reply:
Done